spec/spec_helper.rb

```diff
@@ -7,4 +7,8 @@
 
 Bundler.require :default, :test
 
-RSpec.configure(&:raise_errors_for_deprecations!)
+RSpec.configure do |config|
+  config.raise_errors_for_deprecations!
+
+  config.warnings = true
+end
```

Comment:
@dblock this would enable the warnings and makes also the specs noisy, maybe they could be used for writing better code, but think for that we have rubocop, so by enabling it, we have two of such tools … mmh

if they both give the same or similar warnings, then one is enough and the rubocop messages much more helpful, IMHO the rspec warnings are higher level and helpful for experienced devs, but they also raises the barrier for contributions

if not then it should be disabled